happy/HappyNodeAddress.py

```diff
@@ -187,21 +188,30 @@ def __update_state(self):
             self.removeNodeInterfaceAddress(self.node_id, self.interface, self.ip_address)
 
     def run(self):
-        with self.getStateLockManager():
+        if not self.add and not self.delete:
+            data_state = json.dumps(self.getNodeInterfaceAddresses(self.interface, self.node_id), sort_keys=True, indent=4)
 
-            self.__pre_check()
+            emsg = "virtual node: " + self.node_id + " addresses list for interface id: " + self.interface
 
-            if not self.done:
+            print emsg
```

Comment:
replace `print` with an appropriate log statement. 

It seems to me that the rule of thumb should be that the modules in happy use log statements, command-line convenience scripts use prints.

Reply:
it is used for command-line convenience here, example:
happy-node-address -i BorderRouter -e wlan0
virtual node: BorderRouter addresses list for interface id: wlan0
[
"fd00:0000:fab1:0001:f674:0daa:bdf2:11e9",
"10.0.1.2",
"2001:0db8:0222:0002:5a7c:b6ff:fe44:a118"
]
so I should keep `print` here?

Reply:
This should happen in the CLI tool, not in the module.